### DISCUSSION TITLE
4CH Pro - Phantom button activations

### DISCUSSION THREAD

**Discussion by Sc00by121** [arendst/Tasmota#22700] [Support request]
Hi all,

I have a Sonoff 4CH Pro running Tasmota 14.2.0

At the moment only Relay 1 has something connected to it (boiler) - 2, 3, 4 are empty

For numerous software revisions I have been having issues with phantom activations of the relays.  For a while I thought it was due to user error but on digging deeper into the debug logs it looks to be more of an unexplained issue.

Can someone advise on what might be causing this or whether I can adjust button sensitivity etc? - Is the fact that 3 are not connected to anything possibly causing false activations?

Log entry from today where all 4 were activated at the same time with zero user input.  No timers set etc
[tasmota-log.txt](https://github.com/user-attachments/files/18221835/tasmota-log.txt)

Thank you

**Comment by arendst** [comment]
A quick search on closed issues with keywords "sonoff 4ch ghost" will reveal the most possible cause of your issue.

**Reply by Sc00by121** [reply]
Ah, thank you - I wasn't being specific enough on my searching!

Long story short - 433Mhz receiver looks to be the issue and the solution is to either ground the antenna (apparently doesn't work 100%) or to de-solder the crystal from the board (this is the route I'm going to take) - Only 2 pins to de-solder
